docs/languages/typescript.md

```diff
@@ -245,7 +245,58 @@ Organize imports can also be automatically when you save a TypeScript file by se
     "source.organizeImports": true
 }
 ```
+## Code Actions on Save
 
+The  `editor.codeActionsOnSave` setting lets you configure a set of Code Actions that are run when a file is saved. For example, you can enable organize imports on save by setting:
+```json
+// On save, run both fixAll and organizeImports source actions
+"editor.codeActionsOnSave": {
+    "source.fixAll": true,
+    "source.organizeImports": true,
+}
+```
+You can also enable or disable which Code Actions are run on save per language using a language specific setting:
+```json
+"[typescript]": {
+    "editor.codeActionsOnSave": {
+        "source.organizeImports": true
+    }
+}
+```
+Here are some source actions:
+
+* `"organizeImports"` -  Enables organize imports on save.
+* `"fixAll"` - Auto Fix on Save computes all possible fixes in one round (for all providers including ESLint).
+* `"fixAll.eslint"` -  Auto Fix only for ESLint.
+* `"addMissingImports"` - Adds all missing imports on save.
+
+The TSLint extension already uses `source.fixAll` Code Action kind to implement fix all and auto fix on save.
+
+You can also set `editor.codeActionsOnSave` to an array of Code Actions to execute in order. You can use this to guarantee that a specific Code Action is always run before or after another one that may conflict with it.
+The following `editor.codeActionsOnSave` will always run Organize Imports followed by Fix All once organize imports finishes:
+```json
+"editor.codeActionsOnSave": [
+    "source.organizeImports",
+    "source.fixAll"
+]
+```
+Note: `editor.codeActionsOnSave` supports both object and array configs in settings.
```

Comment:
For me, the documentation explaining the difference between array vs. object was important to understand... So the part above where you say that the array is used *to execute in a specific order* is the most important part. It seems that the explanation is redundant below (maybe‽)